### PR TITLE
[scripts] add minikube to dev_setup

### PR DESCRIPTION
**Pull request by rustielin**
## Motivation

Open sourcing PFN helm chart, which is compatibile with minikube. This is to support tests in CI and also for new developers.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/master/CONTRIBUTING.md#pull-requests)?

yes
## Test Plan

Tested on my Macbook after uninstalling minikube. Also CI (dev-setup-sh-test on alpine, centos, and github)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/diem/diem/tree/master/developers.diem.com, and link to your PR here.)

### CODE REVIEW

File: scripts/dev_setup.sh [hunk 1]
```diff
@@ -151,6 +152,27 @@ function install_helm {
...
+      install_pkg minikube brew
+    else
+      MACHINE=$(uname -m);
+      if [[ $MACHINE == "x86_64" ]]; then
+        MACHINE="amd64"
+      fi
+      TMPFILE=$(mktemp)
+      rm "$TMPFILE"
+      mkdir -p "$TMPFILE"/
+      curl -sL -o "$TMPFILE"/out "https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-$(uname -s | tr '[:upper:]' '[:lower:]')-${MACHINE}"
```
Comment (rexhoffman) [inline scripts/dev_setup.sh:167]: this version of installation would work for mac as well if we want, and could pin the version if we think that's needed.   Not sure it's worth changing though.

Comment (rustielin) [reply scripts/dev_setup.sh:167]: probably not needed. I followed the convention in the other deps we install, e.g. use `brew` when convenient.

**Comment by davidiw** [comment]
Is this still happening? Gonna close PR out, but would love to see this land.

**Comment by rustielin** [comment]
No cycle currently to take a look at it, but yes we should definitely have it. I think at the time, one of the distros didn't like the install

**Comment by rustielin** [comment]
Minikube is installed on each GHA workflow that hits `helm-test`: https://github.com/diem/diem/blob/main/.github/workflows/ci-test.yml#L811

**Pull request closed without merging**